app/components/Exchange/Exchange.jsx

```diff
@@ -361,12 +361,12 @@ class Exchange extends React.Component {
             panelTabs: ws.get("panelTabs", {
                 my_history: 1,
                 history: 1,
-                my_orders: 2,
+                my_orders: 1,
                 open_settlement: 2
             }),
             panelTabsActive: ws.get("panelTabsActive", {
                 1: "my_history",
-                2: "my_orders"
+                1: "my_orders"
```

Comment:
What do you want to achieve here?

Reply:
![image](https://user-images.githubusercontent.com/40874087/75893901-bd454d00-5e33-11ea-94c6-363ad7c2e9a9.png)

To have my trades and my open orders next to market trades.